server/index.js

```diff
@@ -11,42 +13,35 @@ app.use(compression());
 app.use('/:listings_id', express.static(__dirname + '/../client/dist'));
 // Add in add'l parameter to handle 1-100
 
-var bodyParser = require('body-parser');
 app.use(bodyParser.json()); // support json encoded bodies
 app.use(bodyParser.urlencoded({ extended: true })); // support encoded bodies
 
 app.get('/api/listings/:listings_id/questions', (req, res) => {
-  const id = req.params.listings_id; 
+  const id = req.params.listings_id;
   var query = `select * from questions FULL JOIN answers on answers.questionId = questions.id where questions.tripId = ${id}`;
-  db.getAllQuestionsWithTripId(query, (result) => {
+  db.queryDB(query, (result) => {
     res.send(result);
   })
 });
 
-
-app.post('/questions', function (req, res) {
-  // req.body.id or something else will be how I transfer the inputted answer
-  // then I find the specific item with QuestionSet.find(<some parameter>)
-  // I update the answers at that part
-  var newQuestion = QuestionSet.findOneAndUpdate({ primaryRecord: 0 }, { questions: req.body }).exec((err, data) => {
-    //Will also need to update the above line
-    if (err) {
-      console.log(err)
-    } else {
-      res.send(data)
-    }
+app.post('/questions', (req, res) => {
+  const values = [req.body.tripId, `'${req.body.username}'`, `'${req.body.profilepic}'`, `'${req.body.date}'`, `'${req.body.location}'`, req.body.numContributions, req.body.numHelpfulVotes,  `'${req.body.question}'`];
+  const query = `insert into questions (tripId, username, profilePic, date, location, numContributions, numHelpfulVotes, question) 
+  VALUES (${values});`
+  db.queryDB(query, (result) => {
```

Comment:
I would look into params to insert values into your sql string to make sure you're escaping characters (https://node-postgres.com/features/queries)